docs/DeveloperGuide.md

```diff
@@ -408,7 +418,48 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
         Use case resumes at step 2.
 
-    
+**Use case: Find a person**
```

Comment:
There are still trails of the use case for the Find feature from the initial version of the DG in line 357 until 384, which need to be removed.

Reply:
Completely missed that, thanks for bringing it up!